tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -101,7 +101,10 @@ def apply_grad_to_update_var(var, grad):
           var._track_optimizer_slots(_slots)
 
           with ops.control_dependencies([grad]):
-            v0 = var.read_value(do_prefetch=not var.params.bp_v2)
+            if isinstance(var, de.shadow_ops.ShadowVariable):
+              v0 = var.read_value(do_prefetch=False)
```

Comment:
Will it be ok if lookup multiple times in one function call, from different inputs?

Reply:
Hi @Lifann , I remember @MoFHeka has said this was a fake issue?

Reply:
> Hi @Lifann , I remember @MoFHeka has said this was a fake issue, can be close?

Pass it then.